lottie/src/main/java/com/airbnb/lottie/LottieDrawable.java

```diff
@@ -260,6 +265,10 @@ public void setApplyingOpacityToLayersEnabled(boolean isApplyingOpacityToLayersE
     this.isApplyingOpacityToLayersEnabled = isApplyingOpacityToLayersEnabled;
   }
 
+  public void setExtraScaleEnabled(boolean isEnabled) {
```

Comment:
@boduan402 Could you add some documentation around what this does since it is a public API? I'm actually not sure myself when end users would need to use this.

Reply:
I have add doc in latest commit. Thanks!